### PR TITLE
Automatically update flyteidl versions when a github release is created

### PR DESCRIPTION
**Pull request by evalsocket**
Signed-off-by: Yuvraj <evalsocket@gmail.com>

# TL;DR
Automatically update flyteidl versions when a github release is created

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Tracking Issue
https://github.com/lyft/flyte/issues/808

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-__version__ = '0.18.18'
+__version__ = "develop"
```
Comment (EngHabu) [inline setup.py:3]: There is also version.json (for the npm package)

File: .github/workflows/pythonpublish.yaml [hunk 2]
```diff
@@ -17,6 +17,19 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install setuptools wheel twine
+      - name: Bump version and push tag
```
Comment (EngHabu) [inline .github/workflows/pythonpublish.yaml:20]: The trigger for this workflow is a release creation.. I think if we want to automatically bump versions, it should be on master merge...

**Review by EngHabu — COMMENTED** [review]
Awesome! ready to merge?

**Comment by evalsocket** [comment]
@EngHabu Yes...I tested the make command

**Review by EngHabu — APPROVED** [review]

**Pull request merged**